chrome/updater/updater_module.cc

```diff
@@ -98,17 +100,24 @@ void Observer::OnEvent(Events event, const std::string& id) {
       status = "Status is unknown.";
     } else if (crx_update_item_.state == ComponentState::kUpToDate &&
                updater_configurator_->GetPreviousUpdaterStatus().compare(
```

Comment:
Might have brought this up, but since this is a string comparison, using `compare()` is strange (made me feel this was not a string, but a special object with special comparison). So I'd say:

```
    updater_configurator_->GetPreviousUpdaterStatus() == 
        updater_status_string_map.at(UpdaterStatus::kUpdated)
```

is more idiomatic.